ArcGISRuntimeSDKQt_CppSamples/EditData/EditFeatureAttachments/EditFeatureAttachments.qml

```diff
@@ -167,6 +167,7 @@ EditFeatureAttachmentsSample {
             // create the delegate to specify how the view is arranged
             delegate: Item {
                 height: 45
+                width: parent.width
```

Comment:
Small suggestion so it doesn't have to do a parent lookup and may avoid an undefined width error upon instantiation. Otherwise, LGTM
```suggestion
                width: attachmentsList.width
```

Reply:
Ok I switched it to attachmentsList.width